### DISCUSSION TITLE
Specify a reason when kicking an user

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#2506] [Ideas]
### Description

When kicking an user, from a channel or the server, it might be useful to have the ability to type a message to precise the reason to the user.
Also, as a bonus, it might be good to know the user has not just quit the channel or server but has been kicked, with the reason specified as it is the case with IRC.

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [X] iTeamTalk
- [X] TeamTalkServer

**Comment by amirmahdifard** [comment]
i very like this idea, and i hope it will bee there at some point.

**Comment by amirmahdifard** [comment]
@Nardol i think this idea will be added if the developers work on the ban duration feecher at some point, so do you think that we should close this so we don't have 2 tikits for a same topic?

**Comment by amirmahdifard** [comment]
#662

**Comment by Nardol** [comment]
@amirmahdifard  I totally agree with you both are equally useful but the principle is one ticket for one feature/suggestion/bug.
This issue is to specify kicking reason and yours is about ban which is a different feature, the only common point is to specify a reason which IMHO should be another issue than the one you mention which is about ban duration.
I really don't think closing an issue will make both implemented, I think it is another time when being patient is the best to do :slightly_smiling_face: harassing developers won't make things quicker, you only will wast your precious time and flood inbox of people following TeamTalk5 repository, mine for example :laughing:

**Comment by amirmahdifard** [comment]
ok, sorry: i just thort that if they add reezen for ban durations,  this  will also happen: but your true, , so i'm sorry
